### ISSUE TITLE
host.fact.command strips lines

### ISSUE THREAD

**Issue by leahneukirchen** [Fizzadar/pyinfra#393]
**Describe the bug**

`host.fact.command` (and actually anything using a command from FactBase?) removes leading and trailing whitespace of every line.  This makes parsing indentation-sensitive output impossible.

**To Reproduce**

    print(host.fact.command('echo "    whitespace   "'))

**Expected behavior**

it should print `␣␣␣␣whitespace␣␣␣`, not `"whitespace"`.

**Meta**

```

--> Support information:

    If you are having issues with pyinfra or wish to make feature requests, please
    check out the GitHub issues at https://github.com/Fizzadar/pyinfra/issues .
    When adding an issue, be sure to include the following:

    System: Linux
      Platform: Linux-5.7.7_1-x86_64-with-glibc2.2.5
      Release: 5.7.7_1
      Machine: x86_64
    pyinfra: v1.0.4
    Executable: /usr/bin/pyinfra
    Python: 3.8.4 (CPython, GCC 9.3.0)
```

Target is a OpenBSD machine, but I don't think this matters.

**Comment by sysadmin75** [comment]
https://github.com/Fizzadar/pyinfra/pull/396

**Comment by Fizzadar** [comment]
The fix has now been merged for this! Will be included in `1.1` which I'm aiming to release this/next week.

**Comment by Fizzadar** [comment]
1.1 is running a little late but I've dropped `1.1.dev0` which makes this accessible!
